### PR TITLE
SPCTR-3135 Fix: Global - Resolved an issue where a PHP Warning occurred in some cases for some websites.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3135

### Screenshots

- https://d.pr/v/CqNorn

### Types of changes

- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-front-assets.php [hunk 1]
```diff
@@ -179,19 +179,17 @@ public function enqueue_asset_files() {
...
-			or 404 page (which is an obvious case for 404), then get the current page ID and enqueue script.
-			*/
-			if ( ! $cached_wp_query ) {
-				$current_post_assets = new UAGB_Post_Assets( $current_object_id );
+			} else {
+				/*
+				If no posts are present in the category/archive
+				or 404 page (which is an obvious case for 404), then get the current page ID and enqueue script.
+				*/
+				$current_post_assets = new UAGB_Post_Assets( (int) $current_object_id );
```
Comment (adi3890) [inline classes/class-uagb-front-assets.php:192]: why are we type cast $current_object_id now?

Comment (AbhijitNage123) [reply classes/class-uagb-front-assets.php:192]: @adi3890 PHPStan show error for line no 192 i.e Int || null is given and it accept Int only. So to resolved this I type cast it

Comment (adi3890) [reply classes/class-uagb-front-assets.php:192]: @AbhijitNage123 currently we are converting it to integer for every data.
have a check is_int and typecast if required.

Comment (AbhijitNage123) [reply classes/class-uagb-front-assets.php]: Right, Done.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**